tests/api_tests/nodes/test_serializers.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+from urlparse import urlparse
+
+from nose.tools import *  # flake8: noqa
+from dateutil.parser import parse as parse_date
+
+from tests.base import DbTestCase, ApiTestCase, assert_datetime_equal
+from tests.utils import make_drf_request
+from tests.factories import UserFactory, NodeFactory, RegistrationFactory, ProjectFactory
+
+from framework.auth import Auth
+from api.nodes.serializers import NodeSerializer, NodeRegistrationSerializer
+from api.base.settings.defaults import API_BASE
+
+
+class TestNodeSerializer(DbTestCase):
```

Comment:
Correct; they ensure that we don't break API contracts w.r.t. response format without having to do a high-level test like a views test or Runscope test.